test/jdk/sun/security/ssl/SSLSocketImpl/NonAutoClose.java

```diff
@@ -65,37 +58,23 @@ public class NonAutoClose {
     private final static String keyStoreFile = "keystore";
```

Comment:
This isn't in the changeset, but we're trying to move away from the binary {key|trust}store files. We have two template classes, SSLContextTemplate and SSLSocketTemplate, that set up key and trust stores and does the context setup. Can you take a look and see if one of them can be used here?